app/src/Components/Game.js

```diff
@@ -0,0 +1,124 @@
+import * as React from "react";
+
+import * as apiClient from "../apiClient";
```

Comment:
So here I don't think you want `import * as ApiClient from '../apiClient'`, it seems like it would make more sense to explicitly import all the methods, e.g. 

```
import {
  getPlayers,
  getTopPlayers,
  ...
    } from '../apiClient';
```

And then bipass the wrapping you do in your `usePlayers` custom hook.